src/mmw/package.json

```diff
@@ -43,16 +43,16 @@
     "livereload": "0.8.2",
     "lodash": "4.17.19",
     "minifyify": "7.0.0",
-    "mocha": "6.2.2",
+    "mocha": "8.3.0",
```

Comment:
Sorry, had forgotten about this PR!

No we haven't, but given that we [manually compose the vendor bundle](https://github.com/WikiWatershed/model-my-watershed/blob/ecb997dd22d854b4dddfc41272b7b04908dc2f67/src/mmw/bundle.sh#L111-L148), I don't think that moving those out would have an impact on the final bundle size. It would make semantic sense though, and may still be worth it.